high-availability.md

```diff
@@ -0,0 +1,58 @@
+High Availability Kubecost
+==========================
+
+Running kubecost in high availability mode is a feature that relies on multiple Kubecost replica pods implementing the [ETL Bucket Backup](https://raw.githubusercontent.com/kubecost/docs/main/etl-backup.md) feature combined with a Leader/Follower implementation which ensures that there always exists exactly one leader across all replicas. 
```

Comment:
I would add... "Note: HA mode is Available in Kubecost Enterprise today"

Reply:
It is, have a consistent language that you'd like to use?

Reply:
Not really related to style, I just think we should reword to "...is only available in Kubecost Enterprise." If I wanted to go ahead and add this, would I have to make a separate PR?

Reply:
@mbolt35 what would you suggest?

@bstuder99 I think that's fine but I also think calling this out more clearly can help more users be aware. Examples:

https://slack.com/help/articles/360000394286-Audit-logs-on-Enterprise-Grid
https://guide.kubecost.com/hc/en-us/articles/4407595985047-User-Management-SSO-SAML
https://guide.kubecost.com/hc/en-us/articles/4407601815191-ETL-S3-Backup

Reply:
Can you elaborate what you mean by "calling this out more clearly"? I feel a bolded Note added at the top of the page in the links you provided is pretty clear and draws the reader's eye.

Reply:
Yep, that's all I was suggesting and wanted to see what you thought!

Reply:
@bstuder99 I have no problem with you pulling down this branch and modifying it directly!

Reply:
Ok, can you actually explain briefly how I do that? Not super well-versed with modifying PRs in Git.

Reply:
I've gone ahead and added the disclaimer about Enterprise. If that was the only thing left, is this good to merge?